core/App/ehsm_napi.h

```
@@ -92,6 +92,26 @@ typedef struct {
         retString += "}";
         return StringToChar(retString);
 	};
+
+    static int getCode(char* jsonChar){
+        std::string jsonString = jsonChar;
```

if NULL, will be crashed or not ?

---

will be crashed

---

So please return an error code if NULL.

---

Will be fix in next PR.